deploy/terraform/s3.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_s3_bucket" "security_policy_templates" {
+  bucket = "${var.target_name}-security-policy-templates"
+  acl    = "private"
```

Comment:
We should data classification tags.

Here's an example:

```
tags = {
    Project        = var.jupiterone_project
    Classification = var.content_storage_classification
    Description    = "Table of entities managed by JupiterOne"
  }
```

I think `internal` is a good value for classification.